src/DeviceInterface/ExtensionInterface.js

```diff
@@ -65,8 +72,12 @@ class ExtensionInterface extends InterfacePrototype {
         }
     }
 
-    storeUserData (data) {
-        return chrome.runtime.sendMessage(data)
+    storeUserData ({addUserData: {token, userName, cohort}}) {
+        return chrome.runtime.sendMessage({addUserData: {token, userName, cohort}})
```

Comment:
Opinionated: I've destructured out the pieces of data needed so we're explicitly showing what we're sending. It also makes it mirror the corresponding code in other devices and means we can find it when searching for `addUserData`

Reply:
Should we use type annotations instead? Downside of your approach is that we'd need a new release to pass a new piece of data. On the other hand, it's highly likely we'd need a new release anyway because we'd want to do something with this data.